rasa/model.py

```diff
@@ -231,7 +231,7 @@ def unpack_model(
         with tarfile.open(model_file, mode="r:gz") as tar:
             tar.extractall(working_directory)
             logger.debug(f"Extracted model to '{working_directory}'.")
-    except Exception as e:
+    except tarfile.TarError as e:
```

Comment:
Couldn't there be other errors causing a failure to extract model? Wouldn't this change narrow too much the scenarios?

Reply:
They are all subclasses of `tarfile.TarError` — but I will check if something else can be thrown after lunch today.

Reply:
Yes, only the subclasses of `tarfile.TarError` can be thrown BUT there is one case when `ValueError` can be thrown. Because catching generic exception doesn't sound like a good thing to do, I decided to change it to `(tarfile.TarError, ValueError)`